docs/api.md

```diff
@@ -612,6 +615,11 @@ const page = await context.newPage();
 await page.goto('https://example.com');
 ```
 
+#### browser.defaultBrowserContext()
+- returns: <[BrowserContext]>
+
+Returns default browser context. Default browser context cannot be closed.
```

Comment:
Returns the default browser context. The default browser context can not be closed.

Reply:
Done.